src/calendly-widget.ts

```diff
@@ -417,6 +416,7 @@ export default () => (
           return (
             (this.node = document.createElement("iframe")),
             (this.node.src = this.getSource()),
+            (this.node.setAttribute('data-testid', 'calendly-iframe')),
```

Comment:
Can we remove this attribute? I'd prefer not to add additional attributes to the DOM nodes unless they are necessary for the library to work.

Reply:
Yeah, I can remove this.  I was trying to stay true to the best practices for React Testing Library.  I can do some finagling with `querySelector`.